app/src/main/AndroidManifest.xml

```diff
@@ -9,20 +9,20 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".complaint.NoticeActivity"></activity>
+        <activity android:name=".upcomingMealCard"></activity>
```

Comment:
Why this?

Reply:
By mistake i had created that activilty instead of layout file. I deleted that Activity but i thik i did not delete that from android studio. That's why it is there.